src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Solver.hpp

```diff
@@ -148,6 +169,41 @@ namespace AirflowNetwork {
         bool closing_probability(EnergyPlusData &state, Real64 TimeCloseDuration); // function to perform calculations of closing probability
     };
 
+    
+    struct SimulationControl // Basic parameters for AirflowNetwork simulation
+    {
+        enum class Solver
+        {
+            SkylineLU,
+            ConjugateGradient
+        };
+
+        // Members
+        std::string name; // Provide a unique object name
+        ControlType type =
+            ControlType::NoMultizoneOrDistribution; // AirflowNetwork control: MULTIZONE WITH DISTRIBUTION, MULTIZONE WITHOUT DISTRIBUTION, MULTIZONE
+                                                    // WITH DISTRIBUTION ONLY DURING FAN OPERATION, and NO MULTIZONE OR DISTRIBUTION
+        std::string WPCCntr = "SURFACEAVERAGECALCULATION"; // Wind pressure coefficient input control: "SURFACE-AVERAGE CALCULATION", or "INPUT"
+        iWPCCntr iWPCCnt = iWPCCntr::SurfAvg;              // Integer equivalent for WPCCntr field
+        std::string BldgType = "LowRise";                  // Building type: "LOWRISE" or "HIGHRISE" at WPCCntr = "SURFACE-AVERAGE CALCULATIO"
+        std::string HeightOption = "OpeningHeight";        // Height Selection: "ExternalNode" or "OpeningHeight" at WPCCntr = "INPUT"
+        int maximum_iterations = 500;                      // Maximum number of iteration, default 500
+        int InitFlag = 1;                                  // Initialization flag
+        Solver solver = Solver::SkylineLU;
+        Real64 relative_convergence_tolerance = 0.0001; // Relative airflow convergence
+        Real64 absolute_convergence_tolerance = 1e-06;  // Absolute airflow convergence
+        Real64 convergence_acceleration_limit = -0.5;   // Convergence acceleration limit
+        Real64 MaxPressure = 500.0;                     // Maximum pressure change in an element [Pa]
+        Real64 azimuth = 0.0;                           // Azimuth Angle of Long Axis of Building, not used at WPCCntr = "INPUT"
+        Real64 aspect_ratio = 1.0;                      // Ratio of Building Width Along Short Axis to Width Along Long Axis
+        // Real64 DiffP;                          // Minimum pressure difference
+        std::string InitType = "ZeroNodePressures";    // Initialization flag type:
+        bool temperature_height_dependence = false;    // Choice of height dependence of external node temperature
+        bool allow_unsupported_zone_equipment = false; // Allow unsupported zone equipment
+        // "ZeroNodePressures", or "LinearInitializationMethod"
+    };
+
+
```

Comment:
This is the new simulation control object, there's still more to do here, but can tackle that in a later PR.